### ISSUE TITLE
warp patch + dragmfacts (or dragcfacts)

### ISSUE THREAD

**Issue by dbvasconcelos** [bakkeby/patches#11]
When enabling both patches, the pointer warps to the center of the client right after the click, messing the drag.
Is there any way on restack method to avoid the warp call until the button press release?

**Comment by bakkeby** [comment]
Hmm, I had a similar issue with the default `movemouse` and `resizemouse` functions where clients may decide to send a configure request once they detect that they have been moved over to another monitor, resulting in the window unexpectedly resizing when moving from one screen to another.

I worked around that by setting a "control flag" to ignore such configure requests for the duration that I am using those functions.

https://github.com/bakkeby/dwm-flexipatch/commit/a7da48ca3d7342934e287d23a1866f694210a96c

I'd imagine a similar trick could be used either within the `warp` function itself or in the `restack` function to avoid the warp triggering when using the `dragmfact` or `dragcfact` functions.
